src/bootstrap/Cargo.toml

```diff
@@ -90,3 +90,7 @@ debug = 0
 [profile.dev.package]
 # Only use debuginfo=1 to further reduce compile times.
 bootstrap.debug = 1
+
+[patch.crates-io]
+# FIXME: remove this once the `armv7a-vex-v5` target is available on nightly and cc-rs is updated with it
+cc = { git = "https://github.com/rust-lang/cc-rs", rev = "6fe995c675dc1178f570752fa8f6de82913a91dc" }
```

Comment:
Suggestion [BOOTSTRAP-CLEANUP 1/3]: can you open a cleanup issue for this, and then backlink to that issue from this comment?